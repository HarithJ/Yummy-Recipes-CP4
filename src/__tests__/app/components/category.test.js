import React from 'react';
import { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson, { shallowToJson } from 'enzyme-to-json';

import Category from '../../../components/categories/category.js'

describe("category module", () => {
    const category = shallow(<Category />)
    it("renders properly", () => {
      expect(shallowToJson(category)).toMatchSnapshot();
    })

    it("has a function to delete category", () => {
      expect(category.instance().handleDelete(global.event));
    })

})
