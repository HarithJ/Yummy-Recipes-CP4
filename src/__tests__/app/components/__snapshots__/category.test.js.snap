// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`category module renders properly 1`] = `
<div>
  <div
    className="row justify-content-center"
  >
    <div
      className="col-7"
    >
      <h4>
        <Link
          replace={false}
          to="undefined/recipes"
        />
      </h4>
    </div>
    <div
      className="col-auto"
    >
      <button
        className="btn btn-outline-warning rounded-circle"
        onClick={[Function]}
      >
        <i
          className="fas fa-pencil-alt"
        />
      </button>
    </div>
    <div
      className="col-auto"
    >
      <button
        className="btn btn-outline-danger rounded-circle"
        onClick={[Function]}
      >
        <i
          className="far fa-trash-alt"
        />
      </button>
    </div>
  </div>
  <br />
</div>
`;
